drizzlepac/haputils/astrometric_utils.py

```diff
@@ -948,10 +971,28 @@ def extract_sources(img, dqmask=None, fwhm=3.0, kernel=None, photmode=None,
         input array prior to source identification.
     fwhm : float
         Full-width half-maximum (fwhm) of the PSF in pixels.
-    threshold : float or None
+    kernel : ndarray
+        A numpy array representing the PSF to be used for identifying sources.
+        Image will be convolved with this kernel to highlight sources with shapes
+        similar to the kernel.
+        [Starting in v3.6.0] If None is given, a default 2D Gaussian kernel
+        will be generated with `build_gaussian_kernel` and used based on `fwhm`
+        and `source_box` for the size.
+    photmode : str
+        Specification of the observation filter configuration used for the exposure
+        as reported by the 'PHOTMODE' keyword from the PRIMARY header.
+    segment_threshold : ndarray or None
         Value from the image which serves as the limit for determining sources.
         If None, compute a default value of (background+5*rms(background)).
-        If threshold < 0.0, use absolute value as scaling factor for default value.
+    dao_threshold : float, optional
+        [Deprecated] This parameter is not used.  In fact, it now gets computed
+        internally using the `sigma_clipped_bkg()` function which uses the
+        `dao_nsigma` parameter.
+    dao_nsigma : float
+        This number gets used to determine the threshold for detection of point
+        sources.  The threshold gets computed using a simple `mean + dao_nsigma * rms`,
+        where the `mean` and `rms` are computed from the background levels using
+        `astropy.stats.sigma_clipped_stats`.
     source_box : int
         Size of box (in pixels) which defines the minimum size of a valid source.
```

Comment:
Just being picky ... Is this really the size of the box, is it the length of the side of a square box, or is the number of connected pixels which define a source?  OK, I know it is the last item listed.  It is just I tend to think of a box as an n x m quad.  I would clarify this parameter a bit.

Reply:
See whether or not the small change in commit 30b6df6 clarifies the parameter well enough.